nifi-nar-bundles/nifi-elasticsearch-bundle/nifi-elasticsearch-processors/src/main/java/org/apache/nifi/processors/elasticsearch/QueryElasticsearchHttp.java

```diff
@@ -255,6 +325,8 @@ public void onTrigger(final ProcessContext context, final ProcessSession session
                 .evaluateAttributeExpressions(flowFile).getValue() : null;
         final boolean targetIsContent = context.getProperty(TARGET).getValue()
                 .equals(TARGET_FLOW_FILE_CONTENT);
+        final boolean includeQueryInAttrs = context.getProperty(INCLUDE_QUERY_IN_ATTRS).isSet() ?
```

Comment:
CheckStyle violation here, the ? should be on the next line. You can run your Maven build for the nifi-elasticsearch-bundle with the contrib-check profile activated (`-Pcontrib-check`) and it will run a style check for you.